### PR TITLE
Set information at deploying

### PR DESCRIPTION
**Pull request by JohnTitor**
Sets information(version and commit revision) at deploying. You have to increment version number manually, but this isn't a big deal, I think.

deploy process:

* Opens the PR to increment version and update CHANGELOG.md
* When it is merged, CircleCI tries to deploy

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -42,8 +37,18 @@ func main() {
 		c := "Specify the absolute path to the key file. (default: none)"
 		flag.StringVar(&keyFile, "key", "", c)
 	}
+	var printVersion bool
+	{
+		c := "Print current version. (default: false)"
+		flag.BoolVar(&printVersion, "ver", false, c)
```
Comment (k-nasa) [inline main.go]: [imo]
Can I get the version by typing likes ```frau --ver```?

I think ```--version``` or ```-v```  option is better.

Comment (JohnTitor) [reply main.go]: Oh, I think so. I'll fix.

Comment (JohnTitor) [reply main.go]: fixed!

**Comment by KoujiroFrau** [comment]
:pushpin: Commit 5254a3dda661bea04a8412d67b2955bd5ab1753b has been approved by `k-nasa`

**Comment by KoujiroFrau** [comment]
:pushpin: Commit 5254a3dda661bea04a8412d67b2955bd5ab1753b has been approved by `k-nasa`

**Comment by JohnTitor** [comment]
@KoujiroFrau r=k-nasa

**Comment by KoujiroFrau** [comment]
:pushpin: Commit 5254a3dda661bea04a8412d67b2955bd5ab1753b has been approved by `k-nasa`

**Comment by KoujiroFrau** [comment]
:hourglass: 5254a3dda661bea04a8412d67b2955bd5ab1753b has been merged into the auto branch 7811687a8de2676c838282df323490d921967a83

**Comment by KoujiroFrau** [comment]
:tada: The result of what tried to merge this pull request is `success`.

* [Your tests passed on CircleCI!](https://circleci.com/gh/student-kyushu/frau/52?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

**Pull request merged**